libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -59,6 +64,10 @@ AP_GPS_SBF::AP_GPS_SBF(AP_GPS &_gps, AP_GPS::GPS_State &_state,
     _config_last_ack_time = AP_HAL::millis();
 }
 
+AP_GPS_SBF::~AP_GPS_SBF (void) {
```

Comment:
Extra whitespace.

Also inconsistent with the header file, which doesn't include `void` :-P